LICENSE

```diff
@@ -0,0 +1,20 @@
+Copyright (c) 2010, 2011 the protractor-sublime-snippets Project
```

Comment:
expected this to be reverse-geolocation not "the protractor-sublime-snippets Project"